action.yml

```diff
@@ -0,0 +1,30 @@
+# action.yml
+name: 'Encrypted Secrets from YAML'
+description: 'Decrypt and set env vars for git-trackable secrets, inspired by hiera-eyaml'
+inputs:
+  secrets-file:
+    description: |
+      The eyaml file containing secrets
```

Comment:
It looked like a typo to me first until I realized eyaml is encrypted yaml and that is what it stand in heria-eyaml too 

```suggestion
      The encrypted-YAML file containing secrets
```

Reply:
Ignore this though if it is a common-term

Reply:
It's not really, no. Will change.